SU-SOE/soe_profile
# D8CORE-2339: Sync with stanford_profile and fix install bug

# READY FOR REVIEW 

# Summary
- Syncs up with stanford_profile and fixes issue with autoloading namespace

# Needed By (Date)
- Thursday July 9th

# Urgency
- High

# Steps to Test

1. Check out `D8CORE-2339` branch of Leland D8 stack
2. Do a full `composer install` and `blt di`
3. Validate that site installs and works as it should
4. Run `drush si stanford_profile` and ensure that profile installs as well.

# Affected Projects or Products
- SOE, Leland D8 Stack.

# Associated Issues and/or People
- D8CORE-2339
- https://github.com/SU-SWS/acsf-lelandd8/pull/23

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
